nekoyume/Assets/_Scripts/Helper/ShortcutHelper.cs

```diff
@@ -245,13 +248,92 @@ or ItemSubType.MonsterPart
                 .Show(
                     StageType.EventDungeon,
                     worldMap.SharedViewModel.SelectedWorldId.Value,
-                    stageRow.Id,
-                    $"{RxProps.EventDungeonRow?.GetLocalizedName()} {stageRow.Id.ToEventDungeonStageNumber()}",
+                    eventDungeonStageId,
+                    $"{RxProps.EventDungeonRow?.GetLocalizedName()} {eventDungeonStageId.ToEventDungeonStageNumber()}",
                     true);
             Widget.Find<HeaderMenuStatic>()
                 .UpdateAssets(HeaderMenuStatic.AssetVisibleState.EventDungeon);
         }
 
+        /// <summary>
+        /// Check the shortcut of model is available.
+        /// </summary>
+        public static bool CheckConditionOfShortcut(PlaceType type, int stageId = 0)
+        {
+            switch (type)
+            {
+                case PlaceType.EventDungeonStage:
+                    if (RxProps.EventDungeonInfo.Value is not null)
+                    {
+                        return stageId <=
+                               RxProps.EventDungeonInfo.Value.ClearedStageId + 1;
+                    }
```

Comment:
How about this? If you want to compare the `stageId` with a playable stage id.
Sometimes the `RxProps.EventDungeonInfo.Value.ClearedStageId` can be zero. And we should consider the end of stage.

```suggestion
                    var playableStageId = var openedStageId =
                        RxProps.EventDungeonInfo.Value is null ||
                        RxProps.EventDungeonInfo.Value.ClearedStageId == 0
                            ? RxProps.EventDungeonRow.StageBegin
                            : math.min(
                                RxProps.EventDungeonInfo.Value.ClearedStageId + 1,
                                RxProps.EventDungeonRow.StageEnd);
                    return stageId <= playableStageId;
```

Reply:
Sounds good! I didn't think about end of stage. I'll re-write code with considering that.